jujugui/static/gui/src/app/components/usso-login-link/test-usso-login-link.js

```diff
@@ -60,10 +67,17 @@ describe('USSOLoginLink', () => {
           displayType={'button'}
           loginToController={sinon.stub()} />, true);
     assert.deepEqual(component.getRenderOutput(),
-      <juju.components.GenericButton
-        action={component.getMountedInstance()._handleLogin}
-        type="positive"
-        title="Sign up or Login" />);
+      <div className="usso-login">
+        <juju.components.GenericButton
+          action={component.getMountedInstance()._handleLogin}
+          type="positive"
+          title="Sign up or Login" />
```

Comment:
There is an `extraClasses` prop that can be used to add classes to the button. It would be nice if it had the same as the link. Simplifing the hover styling in the component.